src/shared/services/map-location/index.js

```diff
@@ -56,4 +73,36 @@ export const formatAddress = address => {
   return display;
 };
 
+export const serviceResult2Address = ({ straatnaam_verkort, huis_nlt, postcode, woonplaatsnaam }) => ({
+  openbare_ruimte: straatnaam_verkort,
+  huisnummer: huis_nlt,
+  huisletter: '',
+  huisnummertoevoeging: '',
+  postcode,
+  woonplaats: woonplaatsnaam,
+});
+
+export const serviceAttributes = [
+  'id',
+  'weergavenaam',
+  'straatnaam_verkort',
+  'huis_nlt',
+  'postcode',
+  'woonplaatsnaam',
+  'centroide_ll',
+];
+
+export const formatResponse = ({ response }) =>
+  response.docs.map(result => {
+    const { id, weergavenaam, centroide_ll } = result;
+    return {
+      id,
+      value: weergavenaam,
+      data: {
+        location: wktPointToLocation(centroide_ll),
+        address: serviceResult2Address(result),
+      },
+    };
+  });
+
```

Comment:
This implementation is very specific to the PDOK service and cannot be considered generic. I'd choose a name for the function that reflects that. Like `formatPDOKResponse` or something.

Reply:
good point!